### PR TITLE
fix #19 update parse method in Job Class & add a test for it

### LINKED ISSUE #19: 2>&1
2>&1 is being added to existing crons after write is executed.

crontab -l
0,30 * * * * sudo -u www-data TERM=dumb  php /var/www/Desk/app/console desk:news:fetch >> /var/log/cron.log 2>&1 2>&1
30 3 */5 * * sudo -u www-data /var/www/Gifts php app/console main:sync:gifts 1 >> /var/log/cron.log 2>&1

Second command added with your bundle. If add third command - it will add another 2>&1 to all commands

### PR DESCRIPTION
**Pull request by chxj1992**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ composer.lock
 composer.phar
 phpunit.xml
 php-cs-fixer.phar
+.idea/
```
Comment (blaugueux) [inline .gitignore]: This should be add in your global `.gitignore` on your computer.

File: composer.json [hunk 2]
```diff
@@ -1,11 +1,15 @@
 {
-    "name": "yzalis/crontab",
+    "name": "chxj1992/crontab",
```
Comment (blaugueux) [inline composer.json]: Do not change the composer path of the library.

File: composer.json [hunk 3]
```diff
@@ -1,11 +1,15 @@
...
     "type": "library",
     "description": "Create, manage or delete crontab and jobs with php.",
     "keywords": ["crontab", "cron", "cron job"],
     "license": "MIT",
     "authors": [
         {
+            "name": "chxj1992",
```
Comment (blaugueux) [inline composer.json]: Please add your informations inside the file you create not as the author of the projet.

File: tests/Crontab/Tests/JobTest.php [hunk 4]
```diff
@@ -117,4 +117,19 @@ public function testToStringException()
             $this->fail('__toString should not raise an InvalidArgumentException');
         }
     }
+
+    public function testParseJobLine() {
```
Comment (blaugueux) [inline tests/Crontab/Tests/JobTest.php]: Bracket should be on the next line.

**Comment by blaugueux** [comment]
Thanks for your work @chxj1992. Check my comments and everything will be good to merge.

**Comment by chxj1992** [comment]
hi @blaugueux. Sorry for the mistake above I made. I'm not very familiar with it, as it is my first merge request on Github ^_^. And thanks a lot for your suggestion !

File: composer.json [hunk 5]
```diff
@@ -14,6 +14,9 @@
         "php": ">=5.3.0",
         "symfony/process": "~2.1"
     },
+    "require-dev": {
+        "phpunit/phpunit": "4.5.1"
```
Comment (blaugueux) [inline composer.json]: `~4.5` instead of a fixed version.

**Comment by blaugueux** [comment]
Congrats for your first PR!

The last step is to squash your pull request so i can merge only one commit into master branch.

File: composer.json (hunk 5 above)
Comment (chxj1992) [inline composer.json]: alright～

**Comment by chxj1992** [comment]
I would like to do that, but I'm not sure how could I squash my pull request, any suggestions ?

**Comment by chxj1992** [comment]
is it right now ?

**Comment by blaugueux** [comment]
Good job :+1:

**Comment by chxj1992** [comment]
awesome !

**Pull request merged**